src/main.py

```diff
@@ -19,18 +19,23 @@ def do_activate(self):
         self.window = MainWindow(self)
 
     def do_open(self, files, filecount, hint):
-        if filecount == 1:
-            file = files[0]
-            if os.path.exists(file.get_path()):
-                fileFormat = file.get_basename().split(".")[-1]
-                if fileFormat == "iso":
-                    self.window = MainWindow(self, file.get_path())
-                else:
-                    print("Only .iso files.")
-            else:
-                print("File not exists : " + file.get_path())
-        else:
+        if filecount != 1:
             print("Only one file.")
+            return
+        
+        file = files[0]
+
+        if not os.path.exists(file.get_path()):
+            print("File not exists : " + file.get_path())
```

Comment:
Little translation fix (not your fault)
```suggestion
            print("File does not exist: " + file.get_path())
```